lib/lhm/throttler/slave_lag.rb

```diff
@@ -1,5 +1,11 @@
 module Lhm
   module Throttler
+
+    def self.format_hosts(hosts)
+      hosts.map { |host| host.partition(':')[0] }
```

Comment:
yeah.  i actually just moved this code around so it could be shared, it was already part of `SlaveLag`.